tests/api2/test_024_container.py

```diff
@@ -49,6 +49,19 @@ def chart_release(app_name, values):
         finally:
             call("chart.release.delete", app_name, job=True)
 
+
+    def wait_for_containerd(container_id):
+        timeout = 60
+        while True:
+            if container_id not in [container['id'] for container in call('docker.container.query')]:
+                break
+
+            time.sleep(10)
+            timeout -= 10
+            if timeout <= 0:
+                fail(f'{container_id!r} container was not deleted in 60 seconds')
```

Comment:
Wait, we probably don't want to call `fail()` here since that will exit the _entire_ CI run. We just need to mark all tests that depend on this container being deleted to be skipped.

Reply:
@yocalebo it won't fail the entire CI run and just the test in question https://github.com/truenas/middleware/blob/472ca363161b0c2c7d14065847d567027933e613/src/middlewared/middlewared/test/integration/utils/pytest.py#L6

Reply:
@Qubad786 it will fail the whole test suite exactly as the comment says

Reply:
This was already being used and i assumed it was to fail the test in question..apologies did not go through the comment.
Fixing it and it's usages ✅